views/calendar/calendar.js

```diff
@@ -144,6 +144,7 @@ let styles = StyleSheet.create({
     backgroundColor: '#ffffff',
   },
   row: {
+    minHeight: 88,
```

Comment:
What does `minHeight` do to the title in the one-line case? Does the title stay at the top of the row?